### ISSUE TITLE
WCAG 4.1.2: Ensure select element has an accessible name (#roleComp)

### ISSUE THREAD

**Issue by green-ninja** [green-ninja/CORS-Examples#18]
#### Issue
`Ensure select element has an accessible name` ([select-name](https://dequeuniversity.com/rules/axe/4.10/select-name?application=msftAI))
#### Target application
[Favro | Book a Demo](https://www.favrotest.com/book-demo)
#### Element path
#roleComp
#### Snippet
`<select id="roleComp" name="roleComp" data-name="roleComp" required="" class="salesform__selectfield w-select">`
#### How to fix
    Fix any of the following:
      Element does not have an implicit (wrapped) <label>
      Element does not have an explicit <label>
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 138.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
